common/templates/layouts/base.njk

```diff
@@ -11,6 +11,7 @@
 {% from "govuk/components/select/macro.njk"            import govukSelect %}
 {% from "govuk/components/summary-list/macro.njk"      import govukSummaryList %}
 {% from "govuk/components/textarea/macro.njk"          import govukTextarea %}
+{% from "govuk/components/panel/macro.njk"             import govukPanel %}
```

Comment:
This commit probably needs to be moved to before the commit where you use this macro otherwise it would throw an error at that point in time :)